examples/my_first_server.zig

```diff
@@ -256,6 +257,36 @@ pub const Handler = struct {
         };
     }
 
+    // Return autocomplete options when a user types one of the
+    // `triggerCharacters` listed in `server_capabilities`.
```

Comment:
The `triggerCharacters` is about automatically triggering autocomplete. It is still possible to manually trigger completion.
```suggestion
```